examples/tutorials/tutorial-websocket/WS Server Alice.ipynb

```diff
@@ -0,0 +1,187 @@
+{
```

Comment:
<p>I think we should remove the errors</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/4539/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I sh*t this file should go away</p>